SLACK.md

```diff
@@ -0,0 +1,30 @@
+# OPA Slack Guidelines v1.0
+
+The [CNCF Code of
+Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+applies in all areas in addition to these rules:
+
+## Channel Spamming
+
+Outside of the issue queues, this is our best communication medium for projects.
+Please keep your messages on topic for each channel, and please use the
+appropriate channel. Please do not cross-post. Spamming channels with
+unsolicited or frequent messages is not allowed.
+
+## Vendor Advertisements
+
+This workspace is intended for conversations related to Open Policy Agent (OPA)
+projects. Please do not post messages advertising proprietary vendor products, services, webinars, blogs, etc.
```

Comment:
It may be valuable to add a quick note that webinars, blogs, etc are OK to supplement a question but should not be the focus. "Advertising" as used here should cover it, but it couldn't hurt to call out that point some more so users don't get the impression they are completely banned.

Reply:
thanks @jpreese! I haven't included a note but I've tweaked the wording to emphasize that OPA Slack should not be used advertise. If people are concerned we can adjust it in the future.

Reply:
I like that version a lot better. Much more clear to me!